submitted_models/ctu_cras_norlab_absolem_sensor_config_1/src/flipper_control_plugin.cpp

```diff
@@ -237,10 +237,16 @@ class FlipperControlPlugin : public System, public ISystemConfigure, public ISys
     this->cmdPosRel = _msg.data();
   }
 
+  public: void OnCmdPosMaxVel(const msgs::Double &_msg)
+  {
+    this->positionCorrectionMaxVelocity = _msg.data();
```

Comment:
You're right, this could get problematic. I've added the mutex and tested  driving a bit with the robots and I did not get in any kind of deadlock with the new implementation.